src/plopp/backends/matplotlib/image.py

```diff
@@ -175,3 +202,37 @@ def update(self, new_values: sc.DataArray):
             New data to update the mesh values from.
         """
         self._data = new_values
+
+    def format_coord(self, x: float, y: float) -> str:
+        """
+        Format the coordinates of the mouse pointer to show the value of the
+        data at that point.
+
+        Parameters
+        ----------
+        x:
+            The x coordinate of the mouse pointer.
+        y:
+            The y coordinate of the mouse pointer.
+        """
+        out = f'x={x:1.4f}, y={y:1.4f}'
```

Comment:
Does this format in a meaningful manner for very large or very small ranges? Do we need to make the format depend on the min/max of the axis?